pkgs/development/go-modules/generic/default.nix

```diff
@@ -48,6 +48,13 @@ let
 
     GO111MODULE = "on";
 
+    # Disable proxying to checksum database. Since Golang 1.13, go uses
+    # proxy.golang.org as default GOPROXY, and it will cache the latest signed
+    # tree size and tree hash in $GOPATH/pkg/sumdb/<sumdb-name>/latest, hence
```

Comment:
Have you tried simply removing the entire `$GOPATH/pkg/sumdb` directory instead? If we turn off caching, we might end up with a diverging result from Go 1.13 used outside of nixpkgs. Context: https://github.com/kubernetes/test-infra/pull/13918

Reply:
sure, I'm going to try it; but IIUC the test-infra issue was because the jsonpatch retagged the repository

Reply:
Yes, that's correct. This will eventually happen again, but for those who are using Go 1.13 to develop their project will not notice the retag because of the proxy. If we then try to package their software, it will fail the build leading to confusion.

Reply:
I think is now done.